.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Do we need to checkin this file?

Reply:
I'm torn between checking it in or not. If we check it in, the launch for tests is there if you clone it. if you don't check it in, you have to re-generate the file in order for debugging to work.

Reply:
I guess it's fine to check it in then. Not going to harm anyone.